RSFormView/Classes/Models/FormField.swift

```diff
@@ -130,6 +132,28 @@ open class FormField {
     return validationType
   }
   
+  /// The default textContentType for a given fieldType, this will be overriden by setting the FormField's *textContentType*
+  var defaultTextContentType: UITextContentType? {
+    var contentType: UITextContentType?
+    
+    switch fieldType {
+    case .usPhone:
+      contentType = .telephoneNumber
+    case .password:
+      if #available(iOS 11.0, *) {
+        contentType = .password
+      }
+    case .email:
+      contentType = .emailAddress
+    case .fiveDigitZipCode:
+      contentType = .postalCode
+    default:
+      break
```

Comment:
why don't you return nil here, instead of having the `var conentType` at the top, and then returning at the end?

Reply:
Because code climate cries out loud if you return too many times in a function, a rule that we can and should configure

Reply:
Agree, early returns preferred while possible.

Reply:
I agree too, but we need to configure code climate rule first